docs/reference/specifications/in-process-providers.md

```diff
@@ -1,10 +1,6 @@
 # Creating an in-process flagd provider
 
-By default, **flagd** is a remote service that is accessed via **grpc** by a client application to retrieve feature flags.
-Depending on the environment, flagd therefore is usually deployed as a standalone service, e.g. as a Kubernetes Deployment,
-or injected as a sidecar container into the pod running the client application,
-as it is done in the [OpenFeature Operator](https://github.com/open-feature/open-feature-operator).
-An in-process flagd provider, on the other hand, is designed to be embedded into the application and therefore
+An in-process flagd provider, on the other hand, is designed to be embedded into the application, and therefore
```

Comment:
> on the other hand,

I don't think this makes sense anymore without the comparison to the RPC one. I think then K wrote this he was trying to contrast them.